_episodes/01-tidiness.md

```diff
@@ -3,79 +3,101 @@ title: "Data Tidiness"
+- Leave the raw data raw - don’t change it!
+- Export the cleaned data to a text-based format like CSV (comma-separated values) format. This ensures that anyone can use the data, and is required by most data repositories.
 
-- [Sequencing results - excel](http://de.iplantcollaborative.org/dl/d/94749C20-0DA6-45A6-8CA0-11D6A7C75E78/sequencing_results_metadata.xls) **FIXLINK**
-- [Sequencing results - text](./sequencing_results_metadata.txt) **FIXLINK**
+![Messy spreadsheet]((../fig/2_datasheet_example.jpg))
 
-### **Questions**
+> ## Exercise
+> This is some potential spreadsheet data for an experiment. With the person next to you, for about 2 minutes, discuss some of the problems with this spreadsheet data.
+>
+> ![Messy spreadsheet]((../fig/2_datasheet_example.jpg))
+>
+> > A full set of types of issues with spreadsheet data is [here](http://www.datacarpentry.org/spreadsheet-ecology-lesson/02-common-mistakes/). Not all are present in this example. Discuss with the group what they found.
+> >
+> {: .solution}
+{: .challenge}
 
-1. How are these samples organized?
-2. If you wanted to relate file names to the sample names submitted above (e.g. wild type...) could you do so?
-3. What do the \_R1/\_R2 extensions mean in the file names?
-4. What does the '.gz' extension on the filenames indicate?
-5. What is the total file size - what challenges in downloading and sharing these data might exsist?
+## Further notes on data tidiness
 
-## Summary 
+Data organization at this point of your experiment will help facilitate your analysis later, as well as prepare your data and notes for data deposition now often required by journals and funding agencies. If this is a collaborative project, as most projects are now, it's also information that collaborators will need to interpret your data and results and is very useful for communication and efficiency.
 
-Before analysis of data has begun, there are already many potential areas for errors and omissions. Keeping organized and keeping a critical eye can help catch mistakes. 
+Fear not! If you have already started your project, and it's not set up this way, there are still opportunities to make updates. One of the biggest challenges is tabular data that isn't formatted so computers can use it, or has inconsistencies that make it hard to analyze.
 
-One of Data Carpentry's goals is to help you achieve *competency* in working with bioinformatics. This means that you can accomplish routine tasks, under normal conditions, in an acceptable amount of time. While an expert might be able to get to a solution on instinct alone - taking your time, using Google, and asking for help are all valid ways of solving your problems. As you complete the lessons you'll be able to use all of those methods more efficiently.
+There are guidelines on how to structure your data, outlined in our [Data Carpentry Ecology spreadsheet lesson](http://www.datacarpentry.org/OpenRefine-ecology-lesson/)
```

Comment:
I think these guidelines are now included earlier in this episode, so maybe we don't need to reference the Ecology lessons here.